workflow_tests/mocks/testBuildMocks.ts

```diff
@@ -27,19 +27,19 @@ const TESTBUILD__VALIDATEACTOR__SET_HAS_READY_TO_BUILD_LABEL_FLAG__FALSE__STEP_M
 const TESTBUILD__VALIDATEACTOR__TEAM_MEMBER_HAS_FLAG__STEP_MOCKS = [
     TESTBUILD__VALIDATEACTOR__IS_TEAM_MEMBER__TRUE__STEP_MOCK,
     TESTBUILD__VALIDATEACTOR__SET_HAS_READY_TO_BUILD_LABEL_FLAG__TRUE__STEP_MOCK,
-] as const;
```

Comment:
why remove as const? Those arrays aren't supposed to be modified, so that's fine in this file.

UPDATE: it's been causing erorrs, because it lacked `satisfies StepIdentifier[]`